app/src/main/java/edu/cascadia/mobile/apps/movies/MainActivity.java

```diff
@@ -21,24 +22,31 @@
 import edu.cascadia.mobile.apps.movies.model.MovieEntity;
 import edu.cascadia.mobile.apps.movies.ui.MoviesAdapter;
 import edu.cascadia.mobile.apps.movies.utilities.SampleData;
+import edu.cascadia.mobile.apps.movies.viewmodel.MainViewModel;
 
 public class MainActivity extends AppCompatActivity {
 
     RecyclerView mRecyclerView;
     MoviesAdapter mMoviesAdapter;
-    private movieDatabase mDatabase;
+    public movieDatabase mDatabase;
+    //public MainViewModel mViewModel;
+    private List<MovieEntity> movieData = new ArrayList<>();
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+        final MainViewModel mViewModel = ViewModelProviders.of(this).get(MainViewModel.class);
```

Comment:
This is fine to call here, but you still need to create an observable reference between the ViewModel's data and the this view that will update the RecyclerView's data adapter whenever the ViewModel.mMovie data changes.